lib/modules/datasource/conan/index.spec.ts

```diff
@@ -21,11 +25,36 @@ const config: GetPkgReleasesConfig = {
   registryUrls: [nonDefaultRegistryUrl],
 };
 
+const digestConfig: GetDigestInputConfig = {
+  depName: '',
```

Comment:
```suggestion
  depName: 'fake',
```
use fake `depName` should never be empty.